src/loaders/common/loader_helper.py

```diff
@@ -101,20 +109,26 @@ def process_columnar_meta(
         docs: list[dict],
         kbase_collection: str,
         load_ver: str,
+        product_id: str,
+        ignore_missing: bool = False
 ):
     """
     Process the columnar metadata for the genome attributes.
 
     :param docs: the list of documents
     :param kbase_collection: the KBase collection name
     :param load_ver: the load version
+    :param product_id: the product id
+    :param ignore_missing: whether to ignore absent keys in the documents from the column spec file (default: False)
     """
 
-    spec = load_spec(names.GENOME_ATTRIBS_PRODUCT_ID, kbase_collection)
+    spec = load_spec(product_id, kbase_collection)
     columns = list()
     for col_spec in spec.columns:
         key, col_type = col_spec.key, col_spec.type
-        values = _convert_values_to_type(docs, key, col_type)
+        values = _convert_values_to_type(docs, key, col_type, ignore_missing=ignore_missing)
+        if not values:
+            continue
```

Comment:
When might this happen? If for a particular column in the spec there are no keys in any of the pipeline docs?

Reply:
yea. It might happen when all docs are missing the key. But now I am thinking even though that happened, we should still process with the `AttributesColumn`. So removed this.

Reply:
:+1: